.github/workflows/test.yml

```diff
@@ -85,7 +87,7 @@ jobs:
           restore-keys: |
             ${{ runner.OS }}-pods-
 
-      - run: pod install --repo-update
+      - run: pod install --repo-update --verbose
```

Comment:
Not sure we want to have verbose logging on every build

Reply:
There was a terrible bug with no actionable output except "-e script.sh".. so this was helpful for that... I wonder if there's a way to turn it on via a special commit message maybe?

Reply:
I went ahead and removed.